config/example-menu.cfg

```diff
@@ -3,153 +3,116 @@
 # The snippets in this file may be copied into the main printer.cfg file.
 # See the "example.cfg" file for description of common config parameters.
 
-# Available menu elements:
-#   item - purely visual element
-#   command - same like 'item' but with gcode trigger
-#   input - same like 'command' but has value changing capabilities
-#   list - menu element container, with entry and exit gcode triggers
-#   vsdcard - same as 'list' but will append files from virtual sdcard
+# Menu has default variables defined in menu.cfg
+# For example, accessible as printer.menu.default.hotend_target_max
+# Variables can be modified in realtime by using 'Setup -> Edit defaults' menu
+# Following template actions can be used:
+#   printer.menu.action_set_default(name, value) - will set <value> for default variable <name>
+#   printer.menu.action_reset_defaults() - will reset saved user default values to the factory defaults
+# Default variables are accessible from the script as {printer.menu.default.default_variable_name}
+# Don't forget to save your config by using 'Setup -> Save config' after modifing defaults.
+#
+# Menu also has an option for temporary runtime variables.
+# Following template action can be used:
+#   printer.menu.action_set_runtime(name, value) - will set <value> for runtime variable <name>
+# Runtime variables are accessible from the script as {printer.menu.runtime.runtime_variable_name}
+# The runtime variables are not stored between restarts.
+
+# Available options in menu Jinja2 template context:
+# Read-only attributes for menu element:
+#   menu.width - element width (number of display columns)
+#   menu.ns - element namespace
+#   menu.input - input value, only available in input script context
+# List of actions for menu element:
+#   menu.back(force, update) - will execute menu back command, optional boolean parameters <force> and <update>.
+#                      When <force> is set True then it will also stop editing. Default value is False
+#                      When <update> is set False then parent container items are not updated. Default value is True
+#   menu.exit(force) - will execute menu exit command, optional boolean parameter <force> default value False
+#                      When <force> is set True then it will also stop editing. Default value is False
+#   menu.send_event(name, event, [<arg1> [,<arg2> ...]]) - will emit event 'menu:name:event' with args to klippers event system.
+#                      Event first argument is always source element instance
+#   menu.script_by_name(name, from) - other scripts of <from> item, can be included or imported in jinja2.
+#                                     when <from> is not given then default is '.' itself and '..' is parent.
+# Script attributes and methods:
+#   script.event - name of the event that triggered the script, available in script context
+#   script.log_gcode() - will enable script gcode output logging
```

Comment:
Similar comment here - I don't think we should add any menu specific actions.

I can see a use for "scripts".  But, at least for now, I'd limit that to the `render()` capability that is currently in display.py.  That way, interested users only need to learn one Klipper Jinja2 script system.